staging/src/k8s.io/kubectl/pkg/cmd/config/create_cluster.go

```diff
@@ -57,15 +57,18 @@ var (
 		kubectl config set-cluster e2e --certificate-authority=~/.kube/e2e/kubernetes.ca.crt
 
 		# Disable cert checking for the dev cluster entry
-		kubectl config set-cluster e2e --insecure-skip-tls-verify=true`)
+		kubectl config set-cluster e2e --insecure-skip-tls-verify=true
+
+		# Set custom TLS server name to use for validataion for the e2e cluster entry
```

Comment:
...because there's little point in setting the tls server name if you're skipping the whole tls verification step.

Reply:
I suppose these are just examples, showing how to use the commandline?
See above two already existing lines:

```
kubectl config set-cluster e2e --certificate-authority=~/.kube/e2e/kubernetes.ca.crt
...
kubectl config set-cluster e2e --insecure-skip-tls-verify=true
```
They are mutually exclusive ...

Reply:
right, these are distinct examples, not a sequence